masq/src/command_context.rs

```diff
@@ -62,9 +61,10 @@ pub trait CommandContext {
 
 pub struct CommandContextReal {
     connection: ConnectionManager,
-    pub stdin: Box<dyn Read>,
-    pub stdout: Box<dyn Write>,
-    pub stderr: Box<dyn Write>,
+    pub stdin: Box<dyn Read + Send>,
```

Comment:
You were right. It is not necessary anymore. I remember that the compiler prompted me to go this way so that I fulfilled some requirement and by now that obviously passed away. Good.